pyproject.toml

```
@@ -1,15 +1,197 @@
+[project]
+name = "photutils"
+description = "An Astropy package for source detection and photometry"
+requires-python = ">=3.9"
+authors = [
+    { name = "Photutils Developers", email = "photutils.team@gmail.com" },
+]
+keywords = [
+    "astronomy",
+    "astrophysics",
+    "photometry",
+    "aperture",
+    "psf",
+    "source detection",
+    "background",
+    "segmentation",
+    "centroids",
+    "isophote",
+    "morphology",
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Cython",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Astronomy",
+]
+dependencies = [
+    "numpy>=1.22",
+    "astropy>=5.0",
+]
+dynamic = [
+    "version",
+]
+
+[project.readme]
+file = "README.rst"
+content-type = "text/x-rst"
+
+[project.license]
+file = "LICENSE.rst"
+content-type = "text/plain"
+
+[project.urls]
+Homepage = "https://github.com/astropy/photutils"
+
+[project.optional-dependencies]
+all = [
+    "scipy>=1.7.0",
+    "matplotlib>=3.5.0",
+    "scikit-image>=0.19.0",
+    "scikit-learn>=1.0",
+    "gwcs>=0.18",
+    "bottleneck",
+    "tqdm",
+    "rasterio",
+    "shapely",
+]
+test = [
+    "pytest-astropy>=0.10",
+]
+docs = [
+    "sphinx",
+    "sphinx-astropy>=1.6",
+    "scipy>=1.7.0",
+    "matplotlib>=3.5.0",
+    "scikit-image>=0.19.0",
+    "scikit-learn>=1.0",
+    "gwcs>=0.18",
+    "rasterio",
+    "shapely",
+    "tomli; \"python-version\" < \"3.11\"",
```

```suggestion
    "tomli; python-version < '3.11'",
```
I tracked down the remaining RTD failure to this line (it was also preventing editable installs locally).